simtools/db/db_handler.py

```diff
@@ -144,14 +155,15 @@ def get_model_parameters(
 
         # update using simulation model repository
         if self.mongo_db_config.get("db_simulation_model_url", None) is not None:
-            return db_from_repo_handler.update_model_parameters_from_repo(
+            _pars = db_from_repo_handler.update_model_parameters_from_repo(
```

Comment:
This line is not tested (codecov). I would suggest we make sure everything is tested in such a core module

Reply:
Which is actually very weird: all what has been changed is 'return' to an assignment, and suddenly the line is not covered anymore?

Reply:
I've looked at it and decided not to add a dedicated test for two reasons:

- reading the telescope parameters from repo is slow (idea is to use this for local development)
- exactly the same mechanism is implemented in get_site_parameters. Here a test has been added (there are a few site parameters vs 100s of telescope parameters).

Reply:
ok!